torch/testing/_internal/opinfo/definitions/fft.py

```diff
@@ -217,7 +221,18 @@ def mt(shape, **kwargs):
                 precisionOverride({torch.float: 2e-4, torch.cfloat: 2e-4}),
                 "TestFFT",
                 "test_reference_nd",
-            )
+            ),
+            DecorateInfo(
+                toleranceOverride(
+                    {
+                        torch.float16: tol(atol=1e-4, rtol=1e-4),
+                        torch.complex64: tol(atol=1e-4, rtol=1e-4),
```

Comment:
@amjames any idea why this test is failing? 

https://github.com/pytorch/pytorch/actions/runs/3070095254/jobs/4959762987

```
======================================================================
FAIL [0.007s]: test_compare_cpu_fft_hfft2_cuda_float32 (__main__.TestCommonCUDA)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/conda/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 1950, in wrapper
    method(*args, **kwargs)
  File "/opt/conda/lib/python3.10/site-packages/torch/testing/_internal/common_device_type.py", line 378, in instantiated_test
    result = test(self, **param_kwargs)
  File "/opt/conda/lib/python3.10/site-packages/torch/testing/_internal/common_device_type.py", line 853, in dep_fn
    return fn(slf, *args, **kwargs)
  File "/opt/conda/lib/python3.10/site-packages/torch/testing/_internal/common_device_type.py", line 815, in test_wrapper
    return test(*args, **kwargs)
  File "/opt/conda/lib/python3.10/site-packages/torch/testing/_internal/common_device_type.py", line 975, in only_fn
    return fn(slf, *args, **kwargs)
  File "/opt/conda/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 1283, in wrapper
    fn(*args, **kwargs)
  File "/var/lib/jenkins/workspace/test/test_ops.py", line 197, in test_compare_cpu
    self.assertEqual(cpu_result, cuda_result)
  File "/opt/conda/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 2411, in assertEqual
    assert_equal(
  File "/opt/conda/lib/python3.10/site-packages/torch/testing/_comparison.py", line 1093, in assert_equal
    raise error_metas[0].to_error(msg)
AssertionError: Tensor-likes are not close!

Mismatched elements: 3 / 360 (0.8%)
Greatest absolute difference: 2.288818359375e-05 at index (2, 5, 10) (up to 1e-05 allowed)
Greatest relative difference: 5.305602716468591e-05 at index (1, 0, 3) (up to 1.3e-06 allowed)
```
the atol and rtol look to be within the limits I have set here

And it passes on my machine